libs/model/domain.hpp

```diff
@@ -1,30 +0,0 @@
-/*
-Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-     http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-#ifndef IROHA_DOMAIN_HPP
-#define IROHA_DOMAIN_HPP
-
-namespace iroha {
-  namespace model {
-    /**
-     * Domain Model
-     */
-    struct Domain {
```

Comment:
Everyone will be in default domain in current sprint, and we will create this file in the future.